pennylane/plugins/default_gaussian.py

```diff
@@ -836,9 +836,42 @@ def var(self, observable, wires, par):
         _, var = self._observable_map[observable](mu, cov, wires, par, hbar=self.hbar, total_wires=self.num_wires)
         return var
 
+
     def sample(self, observable, wires, par, n=None):
-        raise NotImplementedError("Sampling is not supported in default.gaussian, "
-                                  "please install PennyLane-SF or another plugin capable of sampling")
+        r"""Returns n samples of the given observable in wires.
+
+        Args:
+            observable (string): Observable to be measured
+            wires (int or Sequence[int]): indices of the requested wires
+            par (list): parameters for the observable
+            n (int): number of samples to be generated
+
+        Returns:
+            array: samples
+        """
+        if n is None:
+            n = self.shots
+        if n <= 0 or not isinstance(n, int):
+            raise ValueError("The number of samples must be a positive integer.")
+
+        if len(wires) != 1:
+            raise ValueError("Only one mode can be measured in homodyne.")
+
+        if observable == "X":
+            phi = 0.0
+        elif observable == "P":
+            phi = np.pi/2
+        elif observable == "QuadOperator":
+            phi = par[0]
+        else:
+            raise NotImplementedError("Sampling for this observable is not implemented.")
+        mu, cov = self.reduced_state(wires)
+        rot = rotation(phi)
+        muphi = rot.T @ mu
+        covphi = rot.T @ cov @ rot
+        stdphi = np.sqrt(covphi[0, 0])
+        meanphi = muphi[0]
+        return np.random.normal(meanphi, stdphi, n)
```

Comment:
I'm quite sure that `self.hbar` or a parameter `hbar` has to come in here somewhere.

Reply:
I don't think so. The covariance matrix and vector of means already have units. For instance if the state is vacuum then V = \hbar/2 I. And thus the automatically the variance of x or p will have the scale \sqrt{\hbar/2}

Reply:
I'm a little bit confused -- why do you need to access hbar in `fock_expectation` and `poly_quad_expectations` but not here? Is this because here is a special case where we don't need it?

Reply:
Some gates need it, some don't (usually depends on whether the gates are parameterized in terms of a,a^\dagger or x and p

Reply:
Hey @johannesjmeyer --- It  gets a bit confusing, the reason for this is that the expectation value of low (<= 2) degree polynomials can be obtained directly from integrals in phase space whereas for higher degree operators you need to add something called the Groenewold anomaly which is \hbar dependent. Happy to discuss offline if you are interested :)

Reply:
Thanks for that input! It would be nice if you could point me to a place where I can read about it!